app/routes/docs.faq/route.mdx

```diff
@@ -60,6 +60,14 @@ Clients connecting to GCN only need to be able to make _outbound_ (egress) TCP c
   </tbody>
 </table>
 
+### How do I make a 'cron' job to periodically check for new Kafka messages?
+
+Our [Kafka client setup code sample](/docs/client) is designed to be used as part of a long or indefinitely running program. If you leave it running, our Kafka client code will periodically provide you with the latest received messages, and will recover automatically from transient failures like network connectivity problems or temporary broker outages.
```

Comment:
Periodically makes it seem like this is at some fixed interval, which it is not.  Change "periodically" to "immediately" or "promptly".

Reply:
```suggestion
Our [Kafka client setup code sample](/docs/client) is designed to be used as part of a long or indefinitely running program. If you leave it running, our Kafka client code will promptly provide you with the latest received messages, and will recover automatically from transient failures like network connectivity problems or temporary broker outages.
```